extend.php

```diff
@@ -34,6 +34,7 @@
     new Extend\Locales(__DIR__.'/resources/locale'),
 
     (new Extend\Routes('api'))
+        ->post('/fof/polls', 'fof.polls.create', Controllers\CreatePollController::class)
```

Comment:
Should this maybe be `/posts/{id}/poll` instead ? It makes more sense for how the post ID is provided to the backend. 🤷